python_modules/dagster/dagster/_core/definitions/auto_materialize_reason.py

```diff
@@ -0,0 +1,75 @@
+from enum import Enum
+from typing import NamedTuple
+
+
+class AutoMaterializeResult(Enum):
```

Comment:
What do you think about calling this `AutoMaterializeResultType`? I did a grep for Enum in our codebase, and it seems like they basically all end in either "Type" or "Status".

Reply:
yeah makes sense